### ISSUE TITLE
I use the param from the paper and train 320-vgg16 refinDet  model on voc 2007 && voc 2012 trainval data, the map on 2007 test dataset has only 73%, far away from 80%. could you give some ideas to reach the accuracy in paper? 

### ISSUE THREAD

**Issue by lji72** [sfzhang15/RefineDet#107]

**Comment by sfzhang15** [comment]
@lji72 
Do you use our provided python file to train?

**Comment by lji72** [comment]
1.I use the python  file to generate train net, test net and solver net, but I only train to  73%, now using finetuning, the accuracy achieved 75%. but I do not use pretraining mdoel.

**Comment by lji72** [comment]
@sfzhang15 Does the model must use the batchnorm layer reach the 80% map?

**Comment by sfzhang15** [comment]
@lji72 
VGG16 do not have BN layers.
Why do you not use pretraining mdoel?

**Comment by lji72** [comment]
@sfzhang15 ok. I got it, I use the pretraining model to train the model again.
